core/src/OpenACC/Kokkos_OpenACC_ParallelFor_MDRange.hpp

```diff
@@ -697,7 +697,7 @@ class Kokkos::Impl::ParallelFor<Functor, Kokkos::MDRangePolicy<Traits...>,
       }
     }
     int const async_arg = m_policy.space().acc_async_queue();
-#if 0  // FIXME_OPENACC
+#if 0  // FIXME_OPENACC: OpenACC requires tile size to be constant.
```

Comment:
You mean requires them too be known at compile-time?
If so that is a dead end and we can remove the code path altogether

Reply:
Specwise, yes, it should be compile-time constant. However, NVHPC does not complain this issue, even though other errors occur in the loop with non-constant tile size.